### DISCUSSION TITLE
[solved?] Docker Hub 内 deepin-core 镜像长年未更新

### DISCUSSION THREAD

**Discussion by foresee-io** [linuxdeepin/developer-center#3926] [General | 常规讨论]
[Docker Hub](https://hub.docker.com/u/deepin) 显示，**deepin 官方组织** (?) 上传了 `deepin-core` [镜像](https://hub.docker.com/r/deepin/deepin-core/tags)，但它 updated 6 years ago，版本还停留在 deepin 15 。
- <del>建议上传新版 Docker 镜像。</del>
- Update 7:20 UTC: 新镜像在 [这里](https://hub.docker.com/u/linuxdeepin) ，不过这个认证为 `deepin` 的组织官方是否要维护一下？

<img width="817" alt="2023-03-22_13-32" src="https://user-images.githubusercontent.com/32860378/226811663-45a173c4-6146-419a-a046-ad1d1255ee8e.png">

```
# cat /etc/os-release
PRETTY_NAME="Deepin GNU/Linux (Server 15)"
NAME="Deepin GNU/Linux"
VERSION_ID="15"
VERSION="15 (server)"
ID=deepin
HOME_URL="http://www.deepin.com/"
SUPPORT_URL="http://www.deepin.com/support/"
BUG_REPORT_URL="http://mantis.linuxdeepin.com/"
```

**Comment by BLumia** [comment]
我们目前是这个： https://hub.docker.com/u/linuxdeepin

deepin 那个不知道是谁在维护 = =
